storage/innobase/xtrabackup/src/keyring_components.cc

```diff
@@ -15,39 +15,34 @@ along with this program; if not, write to the Free Software
 Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 
 *******************************************************/
-#include <components/keyrings/common/data_file/reader.h>
-#include <components/keyrings/common/data_file/writer.h>
-#include <components/keyrings/common/json_data/json_reader.h>
-#include <components/keyrings/common/json_data/json_writer.h>
-
 #include <dict0dict.h>
 #include <mysqld.h>
+#include <rapidjson/istreamwrapper.h>
 #include <sql/server_component/mysql_server_keyring_lockable_imp.h>
 #include <sql/sql_component.h>
 #include "backup_copy.h"
 #include "backup_mysql.h"
 #include "common.h"
+#include "my_rapidjson_size_t.h"
+#include "rapidjson/document.h"
 #include "utils.h"
 #include "xtrabackup.h"
 
-using keyring_common::data_file::File_reader;
-using keyring_common::data_file::File_writer;
-
 namespace xtrabackup {
 namespace components {
 
-const char *XTRABACKUP_KEYRING_FILE_CONFIG =
-    "xtrabackup_component_keyring_file.cnf";
+const char *XTRABACKUP_KEYRING_FILE_CONFIG = "component_keyring_file.cnf";
+
+const char *XTRABACKUP_KEYRING_KMIP_CONFIG = "component_keyring_kmip.cnf";
 
-const char *XTRABACKUP_KEYRING_KMIP_CONFIG =
-    "xtrabackup_component_keyring_kmip.cnf";
+const char *XTRABACKUP_KEYRING_KMS_CONFIG = "component_keyring_kms.cnf";
```

Comment:
just to confirm, the server cannot use any other name than these right?

Reply:
Correct, those are also hardcoded on the server side.